components/BillDetails.js

```diff
@@ -1,15 +1,20 @@
 import styled from "styled-components";
-import { StyledSubtitle } from "@/components/Fonts.js";
+import {
+  StyledCaption,
+  StyledHeading,
+  StyledSubtitle,
+} from "@/components/Fonts.js";
 
 function BillDetails({ title, amount, children }) {
   return (
     <>
       <Container>
-        <StyledSubtitle>{title}</StyledSubtitle>
-        <List>{children}</List>
+        <StyledHeading>{title}</StyledHeading>
+        <StyledSubtitle>Total {amount} $ </StyledSubtitle>
+        <List>
+          <StyledCaption>{children}</StyledCaption>
+        </List>
```

Comment:
I do not think that it is necessary to work with a list when you only have one list item. And your `StyledCaption` is a `p` element, but it should be a list item if it inside a list. You should change that. 👀